agent/engine/docker_task_engine_windows_test.go

```diff
@@ -48,3 +54,85 @@ func TestDeleteTask(t *testing.T) {
 
 	taskEngine.deleteTask(task)
 }
+
+func TestCredentialSpecResourceTaskFile(t *testing.T) {
```

Comment:
Can you also add negative test case?

Reply:
Did you update the negative test case before resolving this?

Reply:
A negative exists for the same.